src/System.Net.Mail/ref/System.Net.Mail.cs

```diff
@@ -33,12 +33,12 @@ public sealed partial class AlternateViewCollection : System.Collections.ObjectM
     }
     public partial class Attachment : System.Net.Mail.AttachmentBase
     {
-        public Attachment(System.IO.Stream contentStream, System.Net.Mime.ContentType contentType) : base (default(string)) { }
-        public Attachment(System.IO.Stream contentStream, string name) : base (default(string)) { }
-        public Attachment(System.IO.Stream contentStream, string name, string mediaType) : base (default(string)) { }
-        public Attachment(string fileName) : base (default(string)) { }
-        public Attachment(string fileName, System.Net.Mime.ContentType contentType) : base (default(string)) { }
-        public Attachment(string fileName, string mediaType) : base (default(string)) { }
+        public Attachment(System.IO.Stream contentStream, System.Net.Mime.ContentType contentType) : base (default(System.IO.Stream)) { }
+        public Attachment(System.IO.Stream contentStream, string name) : base (default(System.IO.Stream)) { }
+        public Attachment(System.IO.Stream contentStream, string name, string mediaType) : base (default(System.IO.Stream)) { }
+        public Attachment(string fileName) : base (default(System.IO.Stream)) { }
+        public Attachment(string fileName, System.Net.Mime.ContentType contentType) : base (default(System.IO.Stream)) { }
+        public Attachment(string fileName, string mediaType) : base (default(System.IO.Stream)) { }
```

Comment:
Why is the type changing from 'string' to 'System.IO.Stream' here?  This now doesn't match the actual implementation.